docker-compose.yml

```diff
@@ -44,10 +44,11 @@ services:
         max-size: "400k"
         max-file: "20"
   cardano-db-sync-extended:
-    image: inputoutput/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-5.0.1}
+    image: rhyslbw/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-3e02b9370a122cef068e55ac2887b41acc7587ed}
```

Comment:
Do you mean to use this image?

Reply:
Yes, commit builds are not not available for the `inputoutput` repository. Will be swapped out as part of the release pin